packages/subgraph/manifest/templates/contracts/Organization.template.yaml

```diff
@@ -4,8 +4,12 @@
   network: {{network}}
   source:
     abi: Kernel
+   {{#address}}
```

Comment:
what are those one for? `#`?

Reply:
It's mustache's nomenclature see https://github.com/janl/mustache.js#sections